.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -260,11 +260,14 @@ jobs:
             # interruption in such flows.
             ./cilium-cli connectivity test --include-conn-disrupt-test --conn-disrupt-test-setup
 
-      - name: Upgrade Cilium (${{ matrix.name }})
-        uses: cilium/little-vm-helper@908ab1ff8a596a03cd5221a1f8602dc44c3f906d # v0.0.12
+      - name: Upgrade Cilium & Test (${{ matrix.name }})
+        uses: cilium/cilium/.github/actions/conn-disrupt-test@48023b9f6df4fffa80162f418ae2344d33e36b11
```

Comment:
What's this random SHA? Do we want it to point to this commit? Why? Same question for all others

Reply:
It's because of the security concern raised in this discussion: https://github.com/cilium/cilium/pull/27203#discussion_r1297163842

Reply:
Sure, no problem. on v1.13 branch, do we still need to use `cilium/cilium/.github/actions/conn-disrupt-test@SHA` instead of `./.github/actions/conn-disrupt-test` ?

Reply:
I don't know. As it stands I don't think the SHA has any value at all, because it points to the wrong one anyway => it should be the SHA under test, not the current branch SHA. But as I understand it you had tried to use `${{ inputs.context-ref || github.sha }}` and it wasn't working, and I haven't given it much more thought since then.

Reply:
Since sha doesn't make it any better, let's use the simple option.